pages/getting-started/setup-client/android.md

```diff
@@ -7,3 +7,20 @@ parent: Setup Client
 grand_parent: Getting Started
 ---
 # Android Instructions
+
+Our Android application is available to download from Google Play
+
+[![Download at https://play.google.com/store/apps/details?id=com.geeksville.mesh](https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png)](https://play.google.com/store/apps/details?id=com.geeksville.mesh&referrer=utm_source%3Dgithub-homepage)
```

Comment:
I might swap this out for a smaller icon, or add some CSS scaling in the future.

Reply:
Aye, I wasn't too sure how to change it's size in markdown